### ISSUE TITLE
Provide finer grained grained control over type of data store to create (e.g., ItemFileWriteStore vs. ItemFileReadStore)

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#3537]
As @peller points out, one of the "awkward" pieces of my article is the need to programmatically create and set an ItemFileWriteStore to replace the ItemFileReadStore created by Maqetta.

It would be nice if Maqetta offered some finer grained control over what type of data store it creates.

Perhaps this this goes away depending on if and how #3506 is addressed.

**Comment by peller** [comment]
Actually, it might be related to #3189.  If we call the parser manually and stop using parseOnLoad, we may be able to control the way the data stores get initialized.

**Comment by aerwin** [comment]
True... my statement about #3506 was more if we go to dojo.store, then ItemFileWriteStore and ItemFileReadStore would not be the data stores we create anymore and we'd have to look at the issue from the available dojo.store data stores as to what we create (and how much control to give).

**Comment by JonFerraiolo** [comment]
-> M9 Prio Med

@jhpedemonte

**Comment by jhpedemonte** [comment]
-> M10, assign to @peller for now.
